gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/RTrim.feature

```diff
@@ -19,11 +19,12 @@
 Feature: Step - rTrim()
 
   @GraphComputerVerificationInjectionNotSupported
+  # This verifies both ASCII control space and ideographic space character \u3000 are property trimmed.
   Scenario: g_injectX__feature___test__nullX_rTrim
     Given the empty graph
     And the traversal of
       """
-      g.inject("feature  ", "one test ", null, "", " ").rTrim()
+      g.inject("feature  ", "one test ", null, "", " ", "　abc", "abc　", "　abc　", "　　").rTrim()
```

Comment:
could you add test with `\u3000`?

Reply:
I copied and tested `\u3000` on the test file. You may notice the 4 values I newly injected in the test contains a full-width space, not the ASCII control "regular" half-width space character. It did not work just to write as \u3000 on the Gherkin file.
Is there another proper approach to test unicode in Gherkin ?

Reply:
My bad, I didn't notice that there weren't just spaces there.

Probably simplest solution is to add test cases to [Java tests](https://github.com/apache/tinkerpop/blob/3.7-dev/gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/RTrimGlobalStepTest.java)